src/main/java/org/jenkinsci/remoting/engine/WorkDirManager.java

```diff
@@ -282,13 +282,13 @@ public void setupLogging(@CheckForNull Path internalDirPath, @CheckForNull Path
         }
         
         if (overrideLogPath != null) { // Legacy behavior
-            System.out.println("Using " + overrideLogPath + " as an agent Error log destination. 'Out' log won't be generated");
+            LOGGER.log(Level.INFO, "Using {0} as an agent error log destination; output log will not be generated", overrideLogPath);
```

Comment:
With the removal of `System.out.println()`, do the following lines to `flush()` make any sense?

Reply:
Not sure. @oleg-nenashev added this in https://github.com/oleg-nenashev/remoting/commit/d0abdf6278acc73d783e992f6129517dcdb14b31 when there was a following call to `System.setErr`, which disappeared at some point.